darwin/client.go

```diff
@@ -256,6 +256,11 @@ func (cln *Client) WriteCharacteristic(c *ble.Characteristic, b []byte, noRsp bo
 	return nil
 }
 
+// WriteLongCharacteristic writes a characteristic value that is longer than the MTU to a server. [Vol 3, Part F, 3.4.6]
+func (cln *Client) WriteLongCharacteristic(c *ble.Characteristic, b []byte) error {
+	panic("not implemented")
```

Comment:
Rather than panic, this should return an error. Forcing a panic upon a calling program is bit jarring for the caller to deal with.